---a/net/minecraft/command/CommandException.java
+++b/net/minecraft/command/CommandException.java
@@ -1,19 +1,17 @@
 package net.minecraft.command;
 
-public class CommandException extends Exception
-{
+public class CommandException extends Exception {
+
     private final Object[] errorObjects;
     private static final String __OBFID = "CL_00001187";
 
-    public CommandException(String message, Object ... objects)
-    {
+    public CommandException(String message, Object... objects) {
         super(message);
         this.errorObjects = objects;
     }
 
-    public Object[] getErrorObjects()
-    {
+    public Object[] getErrorObjects() {
         return this.errorObjects;
     }
-}
+}
